all, ci: fix some typos & update `_typos.yml`

koibtw opened the pull request:
turns out that the `extend-ignore-identifiers-re` config field wasn't working properly and the correct way to "ignore" "words" is with `extend-words`

ref: https://github.com/crate-ci/typos/blob/master/docs/reference.md

mertcandav commented on std/sys/sys_windows.jule:
```diff
@@ -76,7 +76,7 @@ const (
 	ERROR_DISK_FULL              = Errno(39)
 	ERROR_NOT_SUPPORTED          = Errno(50)
-	ERROR_FILENAME_EXCED_RANGE   = Errno(206)
+	ERROR_FILENAME_EXCEED_RANGE  = Errno(206)
```
```suggestion
	ERROR_FILENAME_EXCED_RANGE   = Errno(206)
```
This error code is belongs to `std/sys` package, which is provides low level interface to operating system. The operating system error codes are copied from the original API with the exact same name. In this case, it is the Windows API.

See [System Error Codes (0-499)](https://learn.microsoft.com/en-us/windows/win32/debug/system-error-codes--0-499-)

koibtw replied:
fair enough

koibtw committed:
fix some typos & update `_typos.yml`

turns out that the `extend-ignore-identifiers-re` config field wasn't working properly and the correct way to "ignore" "words" is with `extend-words`

ref: https://github.com/crate-ci/typos/blob/master/docs/reference.md

Co-authored-by: mertcandav <mertcandav@outlook.com>

mertcandav reviewed (approved):
LGTM. Thanks for your contribution.
